extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -156,10 +158,10 @@ oppia.directive('imageWithRegionsEditor', [
           };
 
           $scope.getPreviewUrl = function(imageUrl) {
-            return $sce.trustAsResourceUrl(
-              '/imagehandler/' + ExplorationContextService.getExplorationId() +
-              '/' + encodeURIComponent(imageUrl)
-            );
+            var objectUrl = AssetsBackendApiService.getImageUrlForPreview(
+              ExplorationContextService.getExplorationId(),
+              encodeURIComponent(imageUrl));
+            return objectUrl;
```

Comment:
You don't need to create the variable here, just return directly.

Reply:
Done!